generate/convert.go

```diff
@@ -174,7 +174,7 @@ func (g *generator) convertArguments(
 		// names.go) and the selection-set (we use all the input type's fields,
 		// and so on recursively).  See also the `case ast.InputObject` in
 		// convertDefinition, below.
-		goTyp, err := g.convertType(nil, arg.Type, nil, options, queryOptions)
+		goTyp, err := g.convertType(nil, arg.Type, false, nil, options, queryOptions)
```

Comment:
Oh interesting, do you think we don't want to apply this to arguments?  I would think we would; while of course there it's easier to set it seems like a more consistent default would be better.  If you feel this is clearer I'm ok with that.  Either way probably worth an explicit mention in the docs.

Reply:
Hm, yes Interesting. I hadn't really considered allowing arguments, but don't see why not.  Plus, allowing it will simplify the code by removing the need to pass around the `isStructField` argument.